src/cache/cache_las.c

```diff
@@ -1534,9 +1536,16 @@ __wt_find_lookaside_upd(
     if (allocated_bytes != 0)
         __wt_free(session, listp);
 
-    /* Couldn't find a record */
-    if (upd == NULL && ret == 0)
-        ret = WT_NOTFOUND;
+    if (ret == 0) {
+        /* Couldn't find a record. */
+        if (upd == NULL) {
+            ret = WT_NOTFOUND;
+            WT_STAT_CONN_INCR(session, cache_lookaside_read_miss);
+        } else {
+            WT_STAT_CONN_INCR(session, cache_lookaside_read);
+            WT_STAT_DATA_INCR(session, cache_lookaside_read);
```

Comment:
I heard that we should add data source statistics sparingly since they occupy memory for every `dhandle` in the system. This stat is easily the most important so I added this one but I wasn't sure about the others.